pkg/hotplug-disk/hotplug-disk.go

```diff
@@ -30,30 +30,55 @@ import (
 	"kubevirt.io/kubevirt/pkg/util"
 )
 
+const (
+	hotplugDisksKubeletVolumePath = "volumes/kubernetes.io~empty-dir/hotplug-disks"
+)
+
 var (
-	podsBaseDir = filepath.Join(util.HostRootMount, util.KubeletPodsDir)
+	// visible for testing
+	TargetPodBasePath = func(podBaseDir string, podUID types.UID) string {
```

Comment:
Yes it is used in other tests too, removing some duplication. It is not perfect but better.

Reply:
You mean that the tests are assigning a stub instead of the original function?
I could not see that, maybe I missed something. I just saw it used with `hotplugdisk.TargetPodBasePath` which can work with a simple exported function name as well.